modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/expression/MBDecision.java

```diff
@@ -87,8 +127,15 @@ public Expression decisionLessEqualThan(){
      *           ["==", null, null]: boolean
      * @return
      */
-    public Expression decisionEqualTo(){
-        return null;
+    private Expression decisionEqualTo() {
+        // validate the arg list
+        if (json.size() != 3) {
+            throwUnexpectedArgumentCount(EQUALS, 2);
+        }
+        // get the comparables
+        Expression comparable1 = parse.string(json, 1);
+        Expression comparable2 = parse.string(json, 2);
+        return ff.function("mbEqualTo", comparable1, comparable2);
```

Comment:
The same question as for notEqualTo for FilterFunction_equalTo / "equalTo" function.

Reply:
Same as above. They both end up calling `StaticGeometry.equalTo(arg0, arg1)`, which doesn't have the same equality checks as the MB Decision expression contract.